app/controllers/sections_controller.rb

```diff
@@ -60,6 +60,7 @@ def destroy
       if @section.has_students?
         flash[:error] = I18n.t('section.delete.not_empty')
       else
+        @section.section_due_dates.each &:destroy
```

Comment:
Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.